gtda/time_series/embedding.py

```diff
@@ -500,8 +496,8 @@ def resample(self, y, X=None):
             Target.
 
         X : None
-            There is no need for input data, yet the pipeline API requires
```

Comment:
God all those linting changes makes the actual changes impossible to see. Please make a specific PR for linting.

Reply:
Sure, I won't mix up linting in future PRs. But please could you let me off the hook this time, as this was a lot of work already? The changes that are not linting are only in the following files (I've gone though carefully and am not going from memory):
- `doc/modules/time_series.rst` -- for insertion in docs
- ` doc/theory/glossary.tex` -- fixing some mathematics in passing (happy to remove this)
- `examples/lorenz_attractor.ipynb`, `gtda/pipeline.py `, ` gtda/tests/test_pipeline.py` -- to adapt to the fact that `width` is no longer a parameter in `SlidingWindow` and `Labeller`
- `gtda/images/preprocessing.py` -- made `_embed` a static method (happy to remove this)
- ` gtda/time_series/__init__.py` -- to add `MultiTakensEmbedding`
- ` gtda/time_series/_utils.py` -- where `_time_delay_embedding` is
- `gtda/time_series/embedding.py` -- where `MultiTakensEmbedding` is alongside improvements and fixes to `SlidingWindow` and `TakensEmbedding` as described in the PR
- `gtda/time_series/target.py ` -- where `Labeller` has been changed and bug-fixed as described in the PR
- `gtda/time_series/tests/test_embedding.py` where tests have been adapted to the API changes/bug fixes, or created for `MultiTakensEmbedding`
- `gtda/time_series/tests/test_target.py` where tests of `Labeller` have been adapted to the API changes/bug fixes
- `gtda/utils/tests/test_validation.py` where tests for `test_multi_time_series` have been added
- `gtda/utils/validation.py ` where `check_multi_time_series` has been placed

Unfortunately, in 14 or the 23 files the changes are substantial!

Reply:
Wow, thanks for making the effort to make a full list!

Maybe a more sensitive solution would be to make a huge linting PR, merge it and then rebase all other PRs.

Reply:
Right. Unfortunately I cannot remember *how* I made these linting changes any more. If you (honestly) find it difficult to review despite the list above, I will remove the linting changes manually here.